graphql/integration_spec.json

```diff
@@ -121,9 +121,1876 @@
         "url": "a url",
         "url_raw": "a raw url"
       }
+    ],
+    "project_ref": [
```

Comment:
Use made up data for this list. Some of these are private and people aren't supposed to know about them

Reply:
true lol, thanks!

Reply:
i removed all private projects from the seed data

Reply:
Anything with 10gen, mms, or cloudp need to be deleted. I would just make up some - I don't think you need this much test data

Reply:
all 10gen, mms and cloudp removed. I would rather do this approach than create all the seed data by hand TBH #lazy

Reply:
This is fine currently, but in the future you should use like 2-3 documents of made up data. Copying data directly from staging is easy for you but makes your reviewers look through 1000+ lines of stuff to make sure you didn't leak anything sensitive